### PR TITLE
scala: add sbt-derivation

### PR DESCRIPTION
**Pull request by kubukoz**

### CODE REVIEW

**Comment by kubukoz** [comment]
oops, editor did some TOC changes. rolling that back

File: README.md [hunk 1]
```diff
@@ -159,6 +160,10 @@
 * [rust-nix-templater](https://github.com/yusdacra/rust-nix-templater) - Generates Nix build and development files for Rust projects.
 * [rust-overlay](https://github.com/oxalica/rust-overlay) - Pure and reproducible nix overlay of binary distributed Rust toolchains.
 
+### Scala
+
+* [sbt-derivation](https://github.com/zaninime/sbt-derivation) -  mkDerivation for sbt, similar to buildGoModule
```
Comment (houstdav000) [inline README.md]: To fit the "awesome list" style guide, I'd suggest editing this line slightly, adding an "A" and a period, and some pre-formatted blocks:

```text
A `mkDerivation` for sbt, similar to `buildGoModule`.
```

**Comment by kubukoz** [comment]
Fixed punctuation.

**Review by houstdav000 — APPROVED** [review]
Looks like a good addition, and fits style requirements

File: README.md (hunk 1 above)
Comment (kubukoz) [reply README.md]: I copied verbatim from the repo's description, but I'll keep this in mind :)

**Pull request merged**